tool/tele/cli/commands.go

```diff
@@ -86,6 +86,10 @@ type HelmBuildCmd struct {
 	Values *[]string
 	// Pull allows to force-pull Docker images even if they're already present.
 	Pull *bool
+	// UpgradeFrom is a path to the image to build an incremental image off of.
+	UpgradeFrom *string
+	// Diff shows differences between two images without building the image.
+	Diff *bool
```

Comment:
This sounds like a different command tbh

Reply:
I thought about it. On one hand, it does feel like a separate command, but on the other I think the UX with the flag is a little better. I.e. you run tele build in a sort of dry run mode to see what will be packaged, then remove the flag and build the actual image. I don't have strong points in favor of one approach vs another though, @knisbet @bernardjkim what do you guys think?

Reply:
I'm in favor of keeping the diff flag. I agree with @r0mant that this provides a slightly better UX as it is now.